go.mod

```diff
@@ -42,4 +42,9 @@ require (
 	nhooyr.io/websocket v1.8.3
 )
 
+replace (
+	github.com/hyperledger/aries-framework-go/component/storageutil => ./component/storageutil
+	github.com/hyperledger/aries-framework-go/spi => ./spi
+)
```

Comment:
Can someone please confirm the validity of this change?

Reply:
there should be no `replace` statements for storage related packages in the root go.mod

cc @DRK3 @troyronda

Reply:
Agreed @Baha-sk

Reply:
Removed